cabal.project

```diff
@@ -201,16 +202,48 @@ constraints:
   , bech32 == 1.1.3
 
   -- Cardano Node dependencies:
-  , cardano-node == 8.9.2
-  , cardano-api ^>=8.39.2.0
-  , cardano-crypto-class >=2.1.4.0
-  , cardano-crypto-class +secp256k1-support
-  , cardano-slotting ^>= 0.1.2
-  , optparse-applicative-fork >= 0.18.1
-  , ouroboros-network ^>= 0.12
-  , plutus-core ==1.21.0.0
-  , plutus-core -with-cert -with-inline-r
-  , plutus-ledger-api ==1.21.0.0
+  , any.cardano-api ==8.39.3.0
```

Comment:
What does the `any.` do?

Reply:
no idea, I copied over from  the node freeze. If there is no opinion against , I will leave it as it is, as copying like that seems a sane approach. I'm sure some of the constraints are implied by other, but computing the "roots" is not obvious by hand. If there is a tool, I am happy to revisit it